### PR TITLE
[StyleCop] Fix all the warnings on Bot.Connector.Authentication.Tests

### PR DESCRIPTION
**Pull request by ceciliaavila**
## Proposed changes
- Add/remove spacing
- Separate _TestConfigurationRetriever_ class to its own file
- Fix field accessors
- Fix upper/lowercase letters
- Remove unnecessary using statements
- Change file encoding from _UTF-8_ to _UTF-8 with BOM_

### CODE REVIEW

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll**

File: tests/Microsoft.Bot.Connector.Tests/Authentication/TestConfigurationRetriever.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
```
Comment (cleemullins) [inline tests/Microsoft.Bot.Connector.Tests/Authentication/TestConfigurationRetriever.cs]: // Copyright headers

**Comment by fuselabs** [comment]
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.Luis.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.AI.QnA.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.ApplicationInsights.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Azure.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.Dialogs.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Builder.TemplateManager.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Configuration.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Connector.dll**
:heavy_check_mark: No Binary Compatibility issues for **Microsoft.Bot.Schema.dll**

File: tests/Microsoft.Bot.Connector.Tests/Authentication/TestConfigurationRetriever.cs (hunk 1 above)
Comment (ceciliaavila) [reply tests/Microsoft.Bot.Connector.Tests/Authentication/TestConfigurationRetriever.cs]: Done. Thanks!

**Comment by ceciliaavila** [comment]
We also fixed some new StyleCop issues in Bot.Configuration and TestBot projects.

**Pull request merged**